### PR TITLE
Creating login module file for login report scenario - RR-757

### PR DESCRIPTION
**Pull request by TomHumphreyBW**
***Changes***

This PR adds a login module to the bwapi repo which will allow the login report endpoint to be tested.

This is blocking my other work on this scenario so I need this to be merged soon.

https://office.brandwatch.com/jira/browse/RR-757

No features changed in this repo so I assume no need to add test results to this.

### CODE REVIEW

**Comment by jonathanchrisp** [comment]
Ooooo I still have access to this repo! 😄  I don't think this matters as open source. LGTM though @TomHumphreyBW. Hope all is well back in Brandwatchland!

**Comment by TomHumphreyBW** [comment]
Hi @jonathanchrisp good to hear from you. Haha thanks for having a look! :D All good here, hope you're doing alright!

File: lib/bwapi/client/brandwatch/clients/login.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module BWAPI
+  class Client
+    module Brandwatch
+      module Clients
+        # Login module for the brandwatch/clients/login/report endpoint
+        module Login
+          # This endpoint is for super admin users to pull login reports for clients
+          #
+          # TODO: Add parameters documentation
```
Comment (lgrzybowski) [inline lib/bwapi/client/brandwatch/clients/login.rb]: I think leaving comment TODO it's not best idea, for me we can use our internal swagger or look directly in BE code when we want to get parameters, as I kind of sure and know this from my own experience that no one will add them in future after merge that PR :D 
So I think @TomHumphreyBW from my perspective you can add the parameters here or just do not add them at all but remove TODO.

Comment (TomHumphreyBW) [inline lib/bwapi/client/brandwatch/clients/login.rb]: Oh I think I just didn't remove that part (I copied most of this from another file and changed the relevant parts).

OK I can remove it

**Comment by lgrzybowski** [comment]
One just two more cents from my side is that this repo is marked as deprecated... 
Lets talk about it on slack :) @TomHumphreyBW

File: lib/bwapi/client/brandwatch/clients/login.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+module BWAPI
+  class Client
+    module Brandwatch
+      module Clients
+        # Login module for the brandwatch/clients/login/report endpoint
+        module Login
+          # This endpoint is for super admin users to pull login reports for clients
```
Comment (TomHumphreyBW) [inline lib/bwapi/client/brandwatch/clients/login.rb]: Note to self: change this comment so that super admin is not mentioned after comment from Conny

**Comment by TomHumphreyBW** [comment]
Hey @lgrzybowski would you mind just to confirm the changes are OK since I've removed the TODO and the comment about superadmins?

File: lib/bwapi/client/brandwatch/clients/login.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+module BWAPI
+  class Client
+    module Brandwatch
+      module Clients
+        # Login module for the brandwatch/clients/login/report endpoint
+        module Login
+          # This endpoint is for users to pull login reports for clients
+
```
Comment (lgrzybowski) [inline lib/bwapi/client/brandwatch/clients/login.rb]: I dont think you need this whitespace here

**Comment by lgrzybowski** [comment]
there is just one small whitespace to remove and this should be good to go

**Pull request merged**